bin/ember

```diff
@@ -38,5 +38,10 @@ resolve('ember-cli', {
   }).then(function(result) {
     var exitCode = typeof result === 'object' ? result.exitCode : result;
     exit(exitCode);
+  }).catch((err) => {
+    // should not normally reach here
+    // this handles the case where console-ui is broken
+    console.error(err);
```

Comment:
In general we would prefer to not mutate this file. But this may be an exception, @rwjblue  c/d?

Reply:
thanks